app/controllers/cars_controller.rb

```diff
@@ -14,8 +11,13 @@ def index
 
   # GET /cars/1 or /cars/1.json
   def show
-    @chargings = @car.chargings.order(created_at: :desc)
-    @repairs = @car.repairs.order(created_at: :desc)
+    @chargings = @car.chargings.order(created_at: :desc).page(params[:chargings_page]).per(5)
```

Comment:
move that number to the constant

Reply:
or even set it in configuration file as default value

Reply:
Done